demo/AppMode.ts

```diff
@@ -31,6 +32,11 @@ export const ReadWriteModes = [
     AppMode.Virtualized
 ];
 
+export const BasicModes = [
+    ...ReadWriteModes,
+    AppMode.ReadOnly
+];
```

Comment:
Each time we added a mode we were running the standalone tests against an additional mode. In most cases, we want a new mode to test specific aspects of the table not all cases.